Datastore/README.md

```diff
@@ -65,6 +65,17 @@ $entity = $datastore->lookup($key);
 This component is considered GA (generally available). As such, it will not introduce backwards-incompatible changes in
 any minor or patch releases. We will address issues and requests with the highest priority.
 
+### Compatibility with Google App Engine
+
+This component is compatible with PHP projects on Google App Engine in the Standard or Flexible environments. To maximize the performance of datastore operations in your app, we recommend the following configuration:
+1. PHP 7 runtime
+2. Require grpc via Composer
```

Comment:
I don't believe this step is necessary, datastore requires GAX which itself requires the gRPC package.

Reply:
Removed.